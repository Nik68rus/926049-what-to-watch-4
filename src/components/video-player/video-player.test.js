import React from 'react';
import renderer from 'react-test-renderer';

import VideoPlayer from './video-player.jsx';

const movie = {
  preview: `route`,
  src: `source`,
};

const children = <div className="children-component" />;

it(`Video player correctly renders`, () => {
  const {preview, src} = movie;

  const tree = renderer.create(<VideoPlayer
    isPlaying={false}
    src={src}
    poster={preview}
    muted={true}
  >
    {children}
  </VideoPlayer>, {
    createNodeMock: () => {
      return {};
    }
  }).toJSON();
  expect(tree).toMatchSnapshot();
});
