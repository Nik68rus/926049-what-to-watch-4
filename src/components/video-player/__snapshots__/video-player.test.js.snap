// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video player correctly renders 1`] = `
<div
  className="children-component"
/>
`;
